src/parser_data.h

```diff
@@ -409,13 +409,16 @@ enum lyd_type {
  *   - @p op - must be NULL, @p parent points to the operation;
  *   Note that error reply should be parsed as 'yang-data' extension data.
  *
+ * Other @p data_type such as LYD_TYPE_DATA_YANG, LYD_TYPE_RPC_YANG, LYD_TYPE_NOTIF_YANG
+ * and LYD_TYPE_REPLY_YANG have parameters with default meaning.
+ *
```

Comment:
I was thinking something along the lines:
```
- ::LYD_TYPE_RPC_YANG:
- ::LYD_TYPE_NOTIF_YANG:
- ::LYD_TYPE_REPLY_YANG:
  - all the parameters have their default meaning.

- ::LYD_TYPE_DATA_YANG:
  - not accepted by the function.
```
I had to look into the code, the last value returns an invalid argument.